### ISSUE TITLE
Refactor for adapters with non Integer PK

### ISSUE THREAD

**Issue by stepanic** [albertosouza/sails-context#2]
Plugin will work only for adapters which has Integer Primary Key, for example: would not work for MongoDB which for PK has UUID

This code should be refactored:
https://github.com/albertosouza/sails-context/blob/master/lib/index.js#L51-52

**Comment by albertosouza** [comment]
yes, this number test solves my problems with mysql query erros but i think in the context we have the related `waterline model class` so we could use it to validate the id ... but i'm out of time to improve it now

**Comment by albertosouza** [comment]
related https://github.com/albertosouza/sails-context/pull/1

**Comment by stepanic** [comment]
Okay, I need to use your npm module in my app.
Do I have your permission to publish new public npm package from my fork of your repo as npm 'sails-context-loopus'

**Comment by albertosouza** [comment]
We can divide the work in sails-context and i can give you permissions to update sails-context in github and npm if you whant

**Comment by stepanic** [comment]
Yes, please add me as collaborator to this repo and I will update it, thanks!

**Comment by albertosouza** [comment]
Done in github, do one test push.
send your login on npm to add you there too.

**Comment by stepanic** [comment]
stepanic is my npm username, this is npm page:
https://www.npmjs.com/~stepanic

**Comment by albertosouza** [comment]
Done.
